doc/api/fs.markdown

```diff
@@ -101,7 +101,7 @@ Objects returned from `fs.watch()` are of this type.
 ### Event: 'change'
 
 * `event` {String} The type of fs change
-* `filename` {String} The filename that changed (if relevant/available)
+* `filename` {String | Buffer} The filename that changed (if relevant/available)
```

Comment:
In the case of these events, how can I as a user predict whether it will be a string or Buffer?